app/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Étape 1 : Construire l'application React
+FROM node:18 AS build
+WORKDIR /app
+COPY package.json package-lock.json ./
+RUN npm install
+COPY . .
+RUN npm run build
+
+# Étape 2 : Servir avec Nginx
+FROM nginx:alpine
```

Comment:
Good practice to specify the version. For instance: FROM nginx:1.27.3-alpine

List of versiosn here: https://hub.docker.com/_/nginx